docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -0,0 +1,79 @@
+---
+approvers:
+- derekwaynecarr title: Manage HugePages
+---
+
+{% capture overview %}
+
+Kubernetes supports the allocation and consumption of pre-allocated huge pages
+by applications in a pod as an **alpha** feature.  This page describes how users
+can consume huge pages and the current limitations.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+1. Kubernetes nodes must pre-allocate huge pages in order for the node to report
+   its huge page capacity.  A node may only pre-allocate huge pages for a single
+   size.
+1. A special **alpha** feature gate `HugePages` has to be set to true across the
+   system: `--feature-gates="HugePages=true"`.
+
+The nodes will automatically discover and report all huge page resources as a
+schedulable resource.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## API
+
+Huge pages can be consumed via container level resource requirements using the
+resource name `hugepages-{size}`, where size is the most compact binary notation
+using integer values supported on a particular node.  For example, if a node
```

Comment:
Our style is to use angle brackets for placeholders.

`hugepages-<size>`, where <size> is the ...